compiler/ballerina-lang/src/test/java/org/ballerinalang/toml/ManifestProcessorTest.java

```diff
@@ -126,44 +129,115 @@ public void testLicenseDescription() throws TomlException {
         Assert.assertEquals(manifest.getProject().getLicense().get(1), "Apache-2.0");
     }
 
+    // Disabled on Windows
```

Comment:
Shall we change the `description` of the test method appropriately and remove this comment?

Reply:
yes, modified the descriptions accordingly.